robotiq_85_description/urdf/tool_changer.urdf.xacro

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-  <xacro:macro name="tool_changer" params="prefix connected_to">
+  <xacro:macro name="tool_changer" params="prefix connected_to rotation:=^|0">
```

Comment:
Should the default rotation value be `pi/2.0` to match the original yaw value in the joint?

Reply:
Yea, good catch. Will update